frontend/src/views/Apps/AppDetail/AppDetail.js

```diff
@@ -62,7 +63,8 @@ class AppDetail extends Component {
       networkData,
     } = await ApplicationService.getApplication(address);
 
-    const chains = await NetworkService.getNetworkChains(networkData.chains);
+    const allChains = await NetworkService.getAvailableNetworkChains();
+    const chains = allChains.map(addIndex);
```

Comment:
Please, change the service method to encapsulate this behavior, add a parameter called **indexed**